apps/settings/js/panels/feedback_send/feedback_send.js

```diff
@@ -106,13 +110,14 @@ define(function(require) {
         currentSetting['deviceinfo.hardware'];
       this._sendData.locale =
         currentSetting['language.current'];
+      this._sendData.happy = this._isHappy();
```

Comment:
Ahh... ok. I think I see what you're doing now. It's hard because there are like zero comments in this file about what the various functions/methods are doing in the grand scheme of things.